docs/rfc/rfc-003-code-owners.md

```diff
@@ -0,0 +1,86 @@
+# RFC-003: Code Owners
+
+## Introduction
+
+To improve the ownership and accountability of our codebase, we propose assigning code owners to specific parts of the codebase. This initiative aims to streamline code reviews, maintain dependencies, and proactively identify potential issues.
+
+## Problem
+
+### Unmaintained Dependencies
+
+Dependencies in repositories often remain unmaintained, requiring tickets to be created to gain visibility and address them.
+
+### Code Reviews
+
+Code reviews frequently stall awaiting reviewers, necessitating manual assignment of reviewers and chasing reviews on Slack.
+
+### Early Problem Detection
+
+Less volatile parts of the codebase are often overlooked, leading to issues being detected late in the development process.
+
+## Proposal
+
+To address these issues, we propose the introduction of code owners. Code owners are team members that are acting lead maintainer of a specific repository or part of the codebase.
+
+They are responsible for:
+
+-   Reviewing pull requests that affect their assigned code.
+-   Keeping dependencies up to date (or creating tickets for them if they require large refactors).
+-   Increasing the static analysis level.
+-   Identifying improvements and creating tickets for them.
+-   Release management for their assigned code (if applicable).
+
+### Time Investment
+
+Allocate roughly 5% of a developer's capacity each sprint (equivalent to one half-day per sprint) for code owner tasks. This can be done once a month, so a full day every two sprints to work on these tasks. _Sprints should be planned with this in mind; sprint work will take precedence over code-owner tasks._
+
+### Coordination
+
+The scrum events will be used to coordinate the code owner's activities:
+
+-   **Sprint Planning**: Allocate time for code owner tasks.
```

Comment:
Is the 5% implicit? So:
...time for code owner tasks_, beyond the 5%, if any._

Reply:
The 5% can be "banked" (so a full day per month perhaps if that works better for people).

The intention was that larger pieces of work that take longer than a half-day (5%) should be normal tickets that can be prioritised by the team/business.